Library/Homebrew/dev-cmd/create.rb

```diff
@@ -230,8 +230,9 @@ def create_formula
         PyPI.update_python_resources! formula, ignore_non_pypi_packages: true if args.python?
 
         puts <<~EOS
-          Please run the following command before submitting:
+          Please audit and test formula before submitting:
             HOMEBREW_NO_INSTALL_FROM_API=1 brew audit --new #{fc.name}
+            HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source --verbose --debug #{fc.name}
```

Comment:
```suggestion
            HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source --verbose --debug #{fc.name}
            HOMEBREW_NO_INSTALL_FROM_API=1 brew test #{fc.name}
```
That better accords with the "audit and test" wording change you're proposing.

Reply:
Yep. Thanks!